src/EnergyPlus/MixedAir.cc

```diff
@@ -4328,6 +4347,16 @@ namespace MixedAir {
 
 		}
 
+		if ( OAController( OAControllerNum ).CoolCoilFreezeCheck ) {
+			MaxOAFracBySetPoint = min( max( MaxOAFracBySetPoint, 0.0 ), 1.0 );
+			OAController( OAControllerNum ).MaxOAFracBySetPoint = MaxOAFracBySetPoint;
```

Comment:
> I ran the same input file using the source code before merge into develop and got non zero values on 4/1 and 4/2. 
Oh.  So maybe my conflict resolution broke it.  I saved a pdf of the diffs before the merge.  I'll double-check against the current diffs.